linux_os/guide/system/software/integrity/fips/enable_fips_mode/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora
+
+title: Enable FIPS Mode
+
+description: |-
+    To enable FIPS mode, run the following command:
+    <pre>fips-mode-setup --enable</pre>
```

Comment:
I think here it should also be described the state characterized as FIPS enabled. Basically what the OVAL check is veriffyin, that is crypto-policy set to FIPS, kernel parameter, dracut module enabled and `/etc/system-fips` file.
At first look, it was not clear that `fips-mode-setup` would do all these changes.

Reply:
Yes, I agree.

Reply:
Thanks for updating the description. 
The paragraph below, right after the command seems too technical and not necessary in rule description. 
The description of FIPS state in last paragraph should be enough.

Reply:
Agree. The paragraph's below are too much.

Reply:
@yuumasato OK